app/src/main/java/com/nhom4/androidmoviedb/presentation/home/HomeFragment.kt

```diff
@@ -0,0 +1,21 @@
+package com.nhom4.androidmoviedb.presentation.home
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.nhom4.androidmoviedb.core.platform.BaseFragment
+import com.nhom4.androidmoviedb.databinding.FragmentHomeBinding
+
+class HomeFragment : BaseFragment<FragmentHomeBinding>() {
+
+    override fun getViewBinding(inflater: LayoutInflater, container: ViewGroup?): FragmentHomeBinding {
+        return FragmentHomeBinding.inflate(inflater, container, false)
+    }
+
+    override fun initView() {
+        // Setup adapter, recyclerView...
+    }
+
+    override fun initData() {
+        // Gọi API, load dữ liệu...
```

Comment:
```suggestion
        // call API, fetch data ...
```
comment code phải = tiếng anh.